src/test/resources/data/timeSeriesPayload.json

```diff
@@ -42,7 +42,7 @@
             "language":"lucene"
           },
           "label":"Revenue Trousers",
-          "value_template":"XXX",
+          "value_template":"$${{value}}",
```

Comment:
Gatling EL feature to [escape $](https://gatling.io/docs/current/session/expression_el/#escaping)
Otherwise it will look in Session object for the value